app/uk/gov/hmrc/eeittadminfrontend/services/AuthService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eeittadminfrontend.services
+
+import cats.data.Validated
+import play.api.Configuration
+import uk.gov.hmrc.eeittadminfrontend.models.{Email, LoginError}
+
+class AuthService(config: Configuration) {
+
+  def checkUser(email: Email): Validated[LoginError, Unit] = {
```

Comment:
Do we really want to mix `Validated` and `Eithers`? In most places we tend to use either. If we start using one and another we might end up with convertions from one format to another. I would suggest to stick to only one data type representing possible failures. To be discussed:
 
Howver if we would like to use Validated I suggest to use nicer syntax so this code could look like:
```scala
import cats.data._
import cats.syntax.all._ //or cats.syntax.validted._
if(users.contains(email.value)) LoginError("Unauthorised User").invalid else ().valid
```